src/libraries/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -39,7 +39,7 @@
   <ItemGroup Condition="'$(TargetsWindows)' == 'true'">
     <Compile Include="System\Uri.Windows.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsUnix)' == 'true'">
+  <ItemGroup Condition="'$(TargetsAnyOS)' == 'true'">
```

Comment:
Nit: I’m not a huge fan of this property name, but the behavior is correct.

Reply:
Yeah that property name is definitely misleading...

Reply:
It has been present since the old config system, So we need to be careful if we decide to change this one name.